mock-api/db.json

```diff
@@ -1,9 +1,73 @@
 {
-    "posts": [
-      { "id": 1, "title": "json-server", "author": "typicode", "category": "json" },
-      { "id": 2, "title": "text-server", "author": "typicode", "category": "text" }
-    ],
-    "comments": [
-      { "id": 1, "body": "some comment", "postId": 1 }
-    ]
-  }
\ No newline at end of file
+  "posts": [
+    {
+      "id": 1,
+      "title": "json-server",
+      "author": "typicode",
+      "category": "json"
+    },
+    {
+      "id": 2,
+      "title": "text-server",
+      "author": "typicode",
+      "category": "text"
+    }
+  ],
+  "comments": [{ "id": 1, "body": "some comment", "postId": 1 }],
+  "commodity": [
+    {
+      "commodityDisplayName": "Tritium",
+      "station": {
+        "name": "Thoreau Station",
+        "arrivalDistance": null,
+        "maxLandingPadSize": "UNKNOWN",
+        "fleetCarrier": true,
+        "requireOdyssey": null,
+        "planetary": null
+      },
+      "systemName": "HIP 113495",
+      "pricesUpdatedAt": "2023-08-02T00:47:01Z",
+      "supply": 9199,
+      "demand": 1,
+      "buyPrice": 49906,
+      "sellPrice": 49184,
+      "distance": 0
+    },
```

Comment:
i'm behind on the latest updates to the endpoints but currently the GET call is made via:

`const queryString = longStringOfFilters`
`const commodityReq = await fetch('/api/v1/trade/locate-commodity/filter?${queryString}',);`

Seems to me that works fine with the json-server, but that's just one variation. We certainly want to prevent devs from needing to connect to morris if possible when they're developing locally, but we'll do what we can as we go!